runtime/src/node_modules/@sapper/internal/manifest-server.d.ts

```diff
@@ -0,0 +1,29 @@
+export const src_dir: string
+export const build_dir: string
+export const dev: boolean
+export const manifest: Manifest
+
+export type Manifest = {
+  server_routes: ServerRoute[];
+  ignore: RegExp[];
+  root_comp?: {
+    default: SSRComponent<unknown>;
+    preload?: Preload<unknown>;
+  }
+  error: SSRComponent
+  pages: Route[]
+};
+
+export type ServerRoute = {
```

Comment:
Some of these are interfaces, some of these are types (not only in this files, others, too). Most of them feel like they could be interfaces. I would try to do it in a consistent manner.

Reply:
Yes, good call.

Reply:
done. thanks!